src/index.js

```diff
@@ -48,6 +50,23 @@ class Masq {
       masqAppBaseUrl: options.masqAppBaseUrl ? options.masqAppBaseUrl : jsonConfig.masqAppBaseUrl,
       swarmConfig: options.swarmConfig ? options.swarmConfig : jsonConfig.swarmConfig
     }
+
+    this._listenForLoginOrSignout()
+  }
+
+  _listenForLoginOrSignout () {
+    window.addEventListener('storage', async (e) => {
+      if (e.key === CURRENT_USER_INFO_STR) {
+        if (e.newValue === null) {
+          if (this.isLoggedIn()) {
+            await this.signout()
+          }
+        } else if (e.newValue) {
+          await this.connectToMasq()
+          this.eventTarget.dispatchEvent(new Event('logged_in'))
```

Comment:
what about using events named `login`, and `signout` ?

Reply:
I prefered 'logged_in' to indicate that the event was not triggered when the login is tried but when the login is successful